sitecore/9.0.1 rev. 171219 XM1/VS2017/Website/.gitignore

```diff
@@ -0,0 +1,271 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
Can put at the root of the repo

Reply:
Did you consider governance? This repository will end up having different projects with different technologies. File types and folders to be ignored may be different among them. Also, you may want to grant permissions to collaborators based on the demos/technologies to review/approve. Keeping the ".gitignore" and the ".gitattributes" local to projects provides easier governance and less dependencies interactions among projects. For instance, this demo is ignoring a whole folder named "storage", this may not be wanted in other project made by somebody else. Happy to change it if you still prefer it. :-)

Reply:
Sorry, I'm confused. Shall I do the changes? or, will the change request be removed?

Reply:
No, leave as-is. I agree with you to leave it.